hydra/_internal/hydra.py

```diff
@@ -136,13 +136,14 @@ def multirun(
     @staticmethod
     def get_sanitized_hydra_cfg(src_cfg: DictConfig) -> DictConfig:
         cfg = copy.deepcopy(src_cfg)
-        with open_dict(cfg):
-            for key in list(cfg.keys()):
-                if key != "hydra":
-                    del cfg[key]
-        with open_dict(cfg.hydra):
-            del cfg.hydra["hydra_help"]
-            del cfg.hydra["help"]
+        with read_write(cfg):
+            with open_dict(cfg):
+                for key in list(cfg.keys()):
+                    if key != "hydra":
+                        del cfg[key]
```

Comment:
This would a bit more compact:
```suggestion
       with flag_override(["struct", "readonly"], False):
          for key in list(cfg.keys()):
            if key != "hydra":
              del cfg[key]
```